tests/sandbox/linalg/test_linalg.py

```diff
@@ -152,3 +154,96 @@ def test_matrix_inverse_solve():
     node = matrix_inverse(A).dot(b).owner
     [out] = inv_as_solve.transform(None, node)
     assert isinstance(out.owner.op, Solve)
+
+
+def test_cholesky_dot_lower():
+    cholesky_lower = Cholesky(lower=True)
+    cholesky_upper = Cholesky(lower=False)
+
+    L = matrix("L")
+    L.tag.lower_triangular = True
+
+    C = cholesky_lower(L.dot(L.T))
+    f = pytensor.function([L], C)
+
+    if config.mode != "FAST_COMPILE":
+        assert (f.maker.fgraph.outputs[0] == f.maker.fgraph.inputs[0]) or (
+            (o := f.maker.fgraph.outputs[0].owner)
+            and isinstance(o.op, (DeepCopyOp, ViewOp))
+            and o.inputs[0] == f.maker.fgraph.inputs[0]
+        )
```

Comment:
Maybe it's easier to just thest the presense/absense of the `CholeskyOp` node in `f.maker.fgraph.apply_nodes`?

Something like (untested code):
```python
assert len(isinstance(node.op, CholeskyOp) for node in f.maker.fgraph.apply_nodes)) == x
```